backend/fms_core/template_importer/row_handlers/normalization/normalization.py

```diff
@@ -46,8 +46,10 @@ def process_row_inner(self, source_sample, destination_sample, process_measureme
             # Case when nM is given
             elif destination_sample['concentration_nm']:
                 if source_sample_obj.is_library:
+                    # TODO: Adapt so we use avg. library size of a pool
```

Comment:
We'll try to get around without avg library size. The less approximation we introduce the better. If this is too slow for the frontend display we'll have one just for the reverse conversion (just for display/endpoint).